resources/certs/jwt/README.md

```diff
@@ -0,0 +1,97 @@
+# Generate JWT Asymmetric Keys
+
+The JWT Authentication mechanism can accept an asymmetric public key to decrypt tokens.
+The OpenHIM can configure this public key in two ways.
+
+Firstly, the public key literal can be supplied in the config.
+
+Second, a file containing the public key in the `pem` format can be placed in the this directory. The file name can then be specified to the config.
+
+The config can be supplied to the OpenHIM in the following ways:
+
+- Config File
+
+  - Public Key Filename.
+
+    ```json
```

Comment:
lets change this to be `js` instead because the ellipses arent valid JSON and being highlighted in red